packages/socket-mode/src/SocketModeClient.ts

```diff
@@ -275,6 +275,8 @@ export class SocketModeClient extends EventEmitter {
         .transitionTo(State.Disconnecting)
       .on(Event.Failure)
         .transitionTo(State.Disconnected)
+      .on(Event.WebSocketOpen)
+        .ignore()
```

Comment:
My only comment is to add a comment here explaining why we ignore it, as you did in the PR description. Future-us will be grateful for the hint!